public/assets/meetups/2024-09-11-lmt-02/data.json

```diff
@@ -0,0 +1,98 @@
+{
+  "title": "II legnicki meetup technologiczny",
```

Comment:
I would go with `"Legnicki meetup technologiczny #2"`, but please talk with @katarzynanowakblumilk about it.

Reply:
The data I provided was to showcase all funcionality of the website. The data.json should be revised to represent correct information about upcoming meetup.